From e1356f954d38793a30d2a1ef37f2b10fc595948e Mon Sep 17 00:00:00 2001
From: Varsha Maira <vmaira@habana.ai>
Date: Sun, 07 Jul 2024 12:35:32 +0300
Subject: [PATCH] [FSW-418] fs1: remove pcr idx get apis in ppboot

This patch removes the functions to fetch the
TPM PCR index in FS1 ppboot. In FS1, ppboot
will not be involved in extending of TPM PCRs.
SVN and manifest hash will be extended in
arcboot and fit hashes will be extended in
preboot.

Change-Id: I72e4e03f76e2a0c114ca2e5b60ce23914386e7fe
Signed-off-by: Varsha Maira <vmaira@habana.ai>
---

diff --git a/arch/fs1/arc/common.c b/arch/fs1/arc/common.c
index ea3164d..0f1079b 100644
--- a/arch/fs1/arc/common.c
+++ b/arch/fs1/arc/common.c
@@ -403,16 +403,6 @@
 	preboot_pcie_wait_for_link();
 }
 
-uint8_t get_ppboot_pcr_index(void)
-{
-	return TPM2_PPBOOT_PCR_IDX;
-}
-
-uint8_t get_preboot_pcr_index(void)
-{
-	return TPM2_PREBOOT_PCR_IDX;
-}
-
 /* by default IO0 fuse is read */
 int preboot_fuse_read(u32 word, u32 *val)
 {
